index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+
+<body>
+  <div>
+    <h1>Awesome Books</h1>
+
+    <div id="books">
+
+    </div>
+
+    <br>
+    <div>
+      <div>
+        <input type="text" placeholder="Title" name="title" id="t">
+      </div>
+      <br>
+      <div>
+        <input type="text" name="author" placeholder="Author" id="a">
+      </div>
+      <br>
+
+      <button id="add_book">Add</button>
+      <br>
+    </div>
+  </div>
+
+  <script src="js/app2.js"></script>
+  <script>
+    const book = new Book()
+    book.loadPrev()
+
+    document.getElementById('add_book').addEventListener('click', () => {
+      book.addBook()
+    })
+
+    function target(element) {
+      book.removeBook(element.id, element.value)
+    }
+  </script>
```

Comment:
* Kindly move all Js codes into a separate Js file because it makes the code more difficult to read. Code that is difficult to read is difficult to maintain. If you can't easily read it and understand what's going on, you won't be able to easily spot bugs.